src/MIDebugEngine/AD7.Impl/AD7BoundBreakpoint.cs

```diff
@@ -65,7 +74,7 @@ public int Delete()
         // Called by the debugger UI when the user is enabling or disabling a breakpoint.
         int IDebugBoundBreakpoint2.Enable(int fEnable)
         {
-            _enabled = fEnable == 0 ? false : true;
+            Enabled = fEnable == 0 ? false : true;
```

Comment:
I add an issue at https://github.com/Microsoft/MIEngine/issues/390 for not able to disabled only one of the boundbreakpoints